lib/features/popup-menu/PopupMenu.js

```diff
@@ -282,6 +306,48 @@ PopupMenu.prototype._attachContainer = function(container, parent, cursor) {
 };
 
 
+/**
+ * Updates popup style.transform with respect to the config and zoom level.
+ *
+ * @method _updateScale
+ *
+ * @param {Object} container
+ */
+PopupMenu.prototype._updateScale = function(container) {
+  var zoom = this._canvas.zoom();
+
+  var scaleConfig = this._config.scale,
+      minScale,
+      maxScale,
+      scale = zoom,
+      transform = '';
+
+  if (scaleConfig !== true) {
+
+    if (scaleConfig === false) {
+      minScale = 1;
+      maxScale = 1;
+    } else {
+      minScale = scaleConfig.min;
+      maxScale = scaleConfig.max;
+    }
+
+    if (isDefined(minScale) && zoom < minScale) {
+      scale = minScale;
+    }
+
+    if (isDefined(maxScale) && zoom > maxScale) {
+      scale = maxScale;
+    }
+
+  }
+
+  transform = 'scale(' + scale + ')';
```

Comment:
I'd inline this statement directly rather than assigning to an additional variable:

```
setTransform(container, 'scale(...)');
```